Art/User-Business-portfolio/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Meu Portfólio</title>
+  <link rel="stylesheet" href="style.css">
```

Comment:
# Incorrect css file name
Currently the filename is matching the link's href
However this filename is incorrect according to both README requirements and the checklist.

- [ ] Please change your file name accordingly for both the css file name and this href